tutorials/kafka/src/main/resources/META-INF/properties.xml

```diff
@@ -19,7 +19,7 @@
     <value>1</value>
   </property>
   <property>
-    <name>dt.application.Kafka2HDFS.operator.kafkaIn.prop.topics</name>
+    <name>dt.application.Kafka2HDFS.operator.kafkaIn.prop.consumer.topic</name>
```

Comment:
This change needs to be reverted (I think I was looking at the wrong file when we discussed this change earlier)
because the property is in AbstractKafkaInputOperator.java
After this change is reverted, this example runs fine on the 3.4.0 sandbox